Sources/stylus/Message/LinkProcessor.swift

```diff
@@ -18,10 +18,10 @@ struct LinkProcessor {
 
         let matches = text.matches(of: urlRegex)
         return matches.map { match in
-            let url = String(text[match.range])
+            var url = String(text[match.range])
             // Remove trailing punctuation that's likely sentence punctuation
             while let lastChar = url.last, [".", ",", ";", ":", "!", "?", ")", "]", "\""].contains(lastChar) {
-                return String(url.dropLast())
+                url.removeLast()
```

Comment:
I don't think there is a bug, the existing code deletes a **single** comma at the end